src/paulscode/android/mupen64plusae/MenuListView.java

```diff
@@ -214,7 +261,7 @@ public View getGroupView( int groupPosition, boolean isExpanded, View convertVie
                     Context.LAYOUT_INFLATER_SERVICE );
             View view = convertView;
             if( view == null )
-                view = inflater.inflate( R.layout.list_item_two_text_icon, null );
+                view = inflater.inflate( R.layout.list_item_menu, null );
```

Comment:
We can probably merge these and just ignore the extra ImageView on the right side when it's not needed.